### PR TITLE
Implement slider

### PR DESCRIPTION
**Pull request by NathanJang**

### CODE REVIEW

File: src/Components/DistanceSlider.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        this.state = {
+            currentDistance: this._maxDistance / 2
+        }
+    }
+
+    propTypes: {
+        handleDistanceChanged: PropTypes.func.isRequired
+    }
+
+    _maxDistance = 10
```
Comment (jbirori) [inline src/Components/DistanceSlider.js:18]: Maybe this could be stores as a state member?

**Review by jbirori — COMMENTED** [review]
Looks good!

File: src/Components/DistanceSlider.js (hunk 1 above)
Comment (NathanJang) [reply src/Components/DistanceSlider.js:18]: Which one?

Comment (jbirori) [reply src/Components/DistanceSlider.js:18]: the `_maxDistance = 10` variable

**Pull request merged**